lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -38,6 +38,27 @@ class Map
   end
 end
 
+module Google
+  module Auth
+    # Extract project_id in initialize.
+    class ServiceAccountCredentials
+      singleton_class.send(:alias_method, :super_make_creds, :make_creds)
+      def self.make_creds(options = {})
+        json_key_io, scope = options.values_at(:json_key_io, :scope)
```

Comment:
I'm confused when you use `json_key_io` as the variable name. Is this value still a "key" at this point, or would it be better named as `json_io`?

Reply:
This is the IO object that will give us the JSON form of a private key. Actually, the naming comes directly from [google-auth-library-ruby](https://github.com/google/google-auth-library-ruby/blob/master/lib/googleauth/service_account.rb#L56), and I'd like to stay as close as possible to that code.